src/AzureAuth/Ado/TokenFetcher.cs

```diff
@@ -32,7 +32,7 @@ internal static class TokenFetcher
             string prompt,
             TimeSpan timeout)
         {
-            return AADTokenFetcher.AccessToken(
+            return new AADTokenFetcher().AccessToken(

```

Comment:
I'm having a hard time locating the corresponding change to the `AADTokenFetcher`. Am I missing something?

Reply:
`AADTokenFetcher` is a type alias define above for `MSALWrapper.TokenFetcher` :)

Reply:
` using AADTokenFetcher = Microsoft.Authentication.MSALWrapper.TokenFetcher;`

Reply:
Okay that makes sense now. I am excited to us able to TDD the common orchestrator in the AzureAuth project.